doc/man7/provider-signature.pod

```diff
@@ -501,7 +501,11 @@ Section 4 "Security Considerations".  The default value for
 nonce B<k> as defined in FIPS 186-4 Section 6.3 "Secret Number
 Generation".
 
-The FIPS provider does not support deterministic digital signature generation.
+The FIPS provider does not support deterministic digital signature generation
+for DSA.
+
+The FIPS provider support for determinisitic digital signature generation for
```

Comment:
This line should probably go into the history section..

Reply:
Suggest breaking this into:

> The FIPS provider supports deterministic digital signature generation for ECDSA.

here and:

> Deterministic digital signature generation for ECDSA was added to the FIPS provider in OpenSSL 3.6.

in the history section.